check-generated-repo.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+PROJECT_NAME=testproject
+SCRIPT_RELATIVE_DIR=$(dirname "${BASH_SOURCE[0]}") 
+cd $SCRIPT_RELATIVE_DIR 
+ROOT_DIR=$(pwd)
+
+set -e
+
+handle_error () {
+    echo "command \"$BASH_COMMAND\" failed with exit code $?.";
+}
+
+cleanup () { 
+    echo "Cleaning up..."
+    rm -rf $ROOT_DIR/pipeline-$PROJECT_NAME;
```

Comment:
This is a nice touch, but maybe it is nicer not to clean up by default and instead echo the `rm -rf` command the user would run to clean up? 

Also, to be extra safe and start clean, one could

```
rm -rf pipeline-testproject/
```
near the beginning of the script.

I'm currently debugging an issue, maybe related to `sed` verison in `scripts/version-sync.sh`, which is where this comment is coming from.

Reply:
pipeline-testproject/ should also be added to .gitignore. since this repo is missing one, could borrow https://github.com/Unstructured-IO/unstructured-api-tools/blob/main/.gitignore